src/evse_monitor.cpp

```diff
@@ -311,6 +312,22 @@ void EvseMonitor::updateEvseState(uint8_t evse_state, uint8_t pilot_state, uint3
   }
 }
 
+void EvseMonitor::verifyPilot() {
+    // After some state changes the OpenEVSE module compiled with PP_AUTO_AMPACITY will reset to the maximum pilot level, so reset to what we expect
+    _openevse.getCurrentCapacity([this](int ret, long min_current, long max_hardware_current, long pilot, long max_configured_current)
+    {
+      if(RAPI_RESPONSE_OK == ret && pilot != getPilot())
```

Comment:
May need to just check for larger pilot values to handle case where EVSE has loimited the current to a lower value than set.
```suggestion
      if(RAPI_RESPONSE_OK == ret && pilot > getPilot())
```

Reply:
yes probably better. Change made